stdlib/public/Concurrency/MonotonicClock.swift

```diff
@@ -0,0 +1,129 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+import Swift
+
+@_silgen_name("swift_get_time")
+@usableFromInline
+func _getTime(
+  absoluteNanoseconds: UnsafeMutablePointer<UInt64>?,
+  continuousNanoseconds: UnsafeMutablePointer<UInt64>?,
+  realtimeSeconds: UnsafeMutablePointer<Int64>?,
+  realtimeNanoseconds: UnsafeMutablePointer<Int64>?
+)
+
+@available(macOS 9999, iOS 9999, tvOS 9999, watchOS 9999, macCatalyst 9999, *)
+public struct MonotonicClock {
+  public struct Instant: Codable, Sendable {
+    var nanoseconds: UInt64
```

Comment:
Two style nits:

- Per the [Leading Underscore Rule](https://github.com/apple/swift/blob/main/docs/StandardLibraryProgrammersManual.md#the-leading-underscore-rule), declarations that aren't public API need to have a leading underscore in their name.
- We prefer to [explicitly spell out every access modifier](https://github.com/apple/swift/blob/main/docs/StandardLibraryProgrammersManual.md#explicit-access-modifiers) in the stdlib, including `internal`.

```suggestion
    internal var _nanoseconds: UInt64
```